container/go/cmd/digester/digester.go

```diff
@@ -65,8 +66,8 @@ func main() {
 	if *format == "docker" && filepath.Ext(*src) != ".tar" {
 		log.Fatalf("Invalid value for argument -src for -format=docker, got %q, want path to tarball file with extension .tar.", *src)
 	}
-	if *format == "legacy" && filepath.Base(*src) != configFile {
-		log.Fatalf("Invalid value for argument -src for -format=legacy, got %q, want path to %s", *src, configFile)
+	if *format == "legacy" && *configPath == "" {
```

Comment:
Same comment as pusher.go about adding errors when flags that shouldn't be specified are specified.